events_structs.go

```diff
@@ -816,6 +818,22 @@ func (ev *EventDeleteAutoscalerType) String() string {
 	return fmt.Sprintf("Alert deleted about %s on container %s", d.Metric, d.ContainerType)
 }
 
+type EventAddonUpdatedTypeData struct {
+	EventAddon
+	Status            []AddonStatus     `json:"status"`
+	AttributesChanged map[string]string `json:"attributes_changed"`
```

Comment:
Why a hash, what's inside, can you set an example of the payload?

Reply:
It's a mistake, an example of the payload is: `['status', 'app_id', ...]` and not `[status: [...], app_id: [...]]`